tools/nuget/generate_nuspec_for_native_nuget.py

```diff
@@ -66,7 +66,7 @@ def generate_file_list_for_ep(nuget_artifacts_dir, ep, files_list):
                     if child_file.is_file() and child_file.suffix == '.dylib' and not is_versioned_dylib:
                         files_list.append('<file src="' + str(child_file) +
                                           '" target="runtimes/osx.10.14-%s/native"/>' % cpu_arch)
-        for cpu_arch in ['x64', 'aarch64']:
+        for cpu_arch in ['x64', 'arm64']:
```

Comment:
The next line use this variable to match file names.  The tarball published to https://github.com/microsoft/onnxruntime/releases will still be onnxruntime-linux-aarch64-[version].tgz.  We use the content of the tarball to make our CPU nuget package. So the line 69 should continue to use "aarch64"(otherwise it won't find the uncompressed tarball). Then, in the next we have
```python
                if cpu_arch == 'x86_64':
                    cpu_arch = 'x64'
```
Please duplicate the code, and change it for arm.
like,
```python
                if cpu_arch == 'aarch64':
                    cpu_arch = 'arm64'
```

Reply:
I have updated the code. I cannot find the tar file as an asset for the published release, is this as expected (the assets not containing any aarch64 files)?

I saw it in the nuget file for 1.10 but it would have been nice to download it separately rather than extracting it from the nuget.

Reply:
Thank you for letting me know. I just uploaded the missing tarball there.